pancancer-arch-3/src/main/java/info/pancancer/arch3/worker/WorkerRunnable.java

```diff
@@ -88,6 +91,11 @@ public WorkerRunnable(String configFile, String vmUuid, int maxRuns) {
     public WorkerRunnable(String configFile, String vmUuid, int maxRuns, boolean testMode, boolean endless) {
         this.maxRuns = maxRuns;
         settings = Utilities.parseConfig(configFile);
+        
```

Comment:
It's not clear to me, so what is the new behaviour? Maybe edit the description in the pull request